### PR TITLE
suthesh/career-page-static

### PR DESCRIPTION
**Pull request by suthesank**
Changes:

-   building department page

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by ShuvoHabib — DISMISSED** [review]

File: src/pages/careers-2/_layout-components/_open-positions.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                                </JobWrapper>
+                            </PositionLink>
+                        )
+                    })}
+
+                {!hidden && (
+                    <LinkButton
+                        external
+                        mt="35px"
+                        to="https://deriv.zohorecruit.com/jobs/Careers"
```
Comment (prince-deriv) [inline src/pages/careers-2/_layout-components/_open-positions.js:67]: Can we add this link to our constants variables?

**Review by prince-deriv — COMMENTED** [review]
Can you fix the code smells :pray:

**Comment by sean-binary** [comment]
Please compress the images yeah 🙏

File: src/pages/careers-2/_layout-components/_open-positions.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import styled from 'styled-components'
+import { zoho_career_url } from '../../../common/constants'
```
Comment (prince-deriv) [inline src/pages/careers-2/_layout-components/_open-positions.js:3]: can we import this using the resolved path: `common/constants` instead :pray:

**Pull request closed without merging**